.travis.yml

```diff
@@ -20,7 +23,7 @@ cache:
         - $HOME/.composer/cache
 
 install:
-    - travis_retry composer install --no-interaction
+    - travis_retry composer update ${COMPOSER_FLAGS} --no-interaction
```

Comment:
Also, no composer.lock is in git, so update and install will behave the same here.